apps/studio/src/features/dashboard/SiteList.tsx

```diff
@@ -1,25 +1,131 @@
 import NextLink from "next/link"
-import { Card, CardHeader, SimpleGrid, Skeleton } from "@chakra-ui/react"
+import {
+  Card,
+  CardHeader,
+  Flex,
+  SimpleGrid,
+  Skeleton,
+  Text,
+} from "@chakra-ui/react"
 import { Link } from "@opengovsg/design-system-react"
 
 import { withSuspense } from "~/hocs/withSuspense"
 import { trpc } from "~/utils/trpc"
 
+const NoResultIcon: React.FC = () => {
```

Comment:
sorry o be annoying, but can we shift this into `components/Svg` or `components/Icon`? also, we need to pass `props: SVGProps<SVGSVGElement>` anad then spread it over the `svg` to allow customisation!